migrations/helpers/DatabaseTaskTemplate.js

```diff
@@ -0,0 +1,67 @@
+class DatabaseTaskTempalte {
+	constructor({
+		id,
+		_id,
+		$set,
+		set,
+		$unset,
+		unset,
+		isModified = true,
+	}) {
+		this.id = _id || id;
+		this.set = $set || set || {};
+		this.unset = $unset || unset || {};
+		this.isModified = isModified;
+	}
+
+	setLog(log) {
+		this.log = log;
+	}
+
+	getId() {
```

Comment:
use `get Id()` getter instead https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Functions/get

Reply:
Do not work in this case, because this.get and this.id is already defined. Only with rename all.